### PR TITLE
[E2E Tests] Add Camel Routes Chart and Debug tests

### PR DESCRIPTION
**Pull request by jsolovjo**
In this PR added:

- Camel Routes Chart test scenario
- Camel Routes basic tests in Debug tab
- Minor fixes and stabilizations

### CODE REVIEW

File: tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/common/CamelChart.java [hunk 1]
```diff
@@ -87,7 +90,7 @@ public void checkSpecificAttributeIsNotDisplayed(String attributeName) {
      * @param value     string value of tested attribute
      */
     public void checkStringAttributeValue(String attribute, String value) {
-        Assertions.assertEquals(getValue(attribute), value);
+        Assertions.assertEquals(value, getValue(attribute));
```
Comment (tadayosi) [inline tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/common/CamelChart.java:93]: Maybe you can rename the method args to make it more visible which one is expected and which one is the value to test.

**Review by tadayosi — APPROVED** [review]

**Comment by tadayosi** [comment]
Also, the tests are failing.

**Comment by jsolovjo** [comment]
> Also, the tests are failing.

Oh yes, I see. Locally it worked fine. So I will check and fix it, thanks

File: tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/common/CamelChart.java (hunk 1 above)
Comment (jsolovjo) [reply tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/fragments/camel/tabs/common/CamelChart.java:93]: Sure, I will rename it. Thanks

**Comment by jsolovjo** [comment]
Hi @tadayosi 
Perhaps, I found an issue - does Debug feature work in Quarkus? From what I see it's missing the Debug tab

**Comment by jsolovjo** [comment]
@tadayosi hello, it seems it's finally got stabilized. Can you check it one more time, please? Thanks

**Comment by tadayosi** [comment]
Great, thanks!

**Pull request merged**